docs/source/tools_integration/ipython.md

```diff
@@ -255,16 +255,16 @@ def some_action():
 * Add the `node` tag to the cell containing your function
 ![Add the node tag graphic](../meta/images/jupyter_notebook_workflow_tagging_nodes.png)
 
-```eval_rst
-.. tip:: The Notebook can contain multiple functions tagged as ``node``, each of them will be exported into the resulting Python file
+```{note}
+The Notebook can contain multiple functions tagged as ``node``, each of them will be exported into the resulting Python file
 ```
 
 * Save your Jupyter Notebook to `notebooks/my_notebook.ipynb`
 * Run `kedro jupyter convert notebooks/my_notebook.ipynb` from the terminal to create a Python file `src/<package_name>/nodes/my_notebook.py` containing `some_action` function definition
 
 
-```eval_rst
-.. tip:: You can also convert all your Notebooks at once by calling ``kedro jupyter convert --all``.
+```{note}
```

Comment:
From documentation contributor guidelines: "Prefer to use `note`, `warning` and `important` only, rather than a number of different colours/types of callout." I tried to color match to the appropriate level.